takeshape/penny
# Add route change loader

### Test Plan (Steps to test):

1. You'll probably want to test locally because the deployment is too fast to observe
1. Change routes
1. Verify that the loader displays
2. This needs a design pass but I think that falls outside the scope of this PR

### Checklist:
